### DISCUSSION TITLE
How do you increase the size of text inside a code block?

### DISCUSSION THREAD

**Discussion by todor-a** [slidevjs/slidev#1443] [Q&A] [answered]
I have a slide that shows a very simple code block but it stretches and takes up the whole space. How do you make the code block take up more space?

**Comment by kermanx** [comment]
You can specify the class or style like this:

```
'''ts {1|2|3}{class:'w-20 children:h-15'}
code
'''
```

**Reply by todor-a** [reply]
I can't seem to get this to work. Also couldn't find an example in the docs. Would you be able to provide one?

**Reply by kermanx** [reply]
The above example looks like this:
![image](https://github.com/slidevjs/slidev/assets/63178754/f09b11b0-a25d-4d71-b202-95473ccf26fb)

It this what you want?

**Reply by todor-a** [reply]
Sorry, it didn't work without the `{1|2|3}`. I added a `{*}` and now the styles are applied. However, this just modifies the width and height of the code block. It does not affect the font of the actual code in the block. I tried `{class:'children:text-2xl'}` but that did not work as well.

**Reply by kermanx** [reply, marked as answer]
You can try `!children:text-2xl`

**Reply by todor-a** [reply]
Yes! That did it.

Do you know if this is in the docs somewhere? I would be happy to add it.

**Reply by kermanx** [reply]
There aren't docs for this now, since this is only CSS-related and not a feature provided by Slidev. It would be nice if we could use classes like `!text-2xl` directly instead of `!children:text-2xl`

**Reply by todor-a** [reply]
Do you think it's worth opening a feature request?

**Reply by kermanx** [reply]
Both OK

**Reply by todor-a** [reply]
[Done](https://github.com/slidevjs/slidev/issues/1449). Thank you once again.
